src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -874,7 +875,19 @@ api_info_header_value(#base_state{}=State) ->
 %% version information (useful for maintaining back-compatibility) as well as OTP version
 %% information (more useful for debugging purposes).
 add_api_info_header(Req, State) ->
-    wrq:set_resp_header("X-Ops-API-Info", api_info_header_value(State), Req).
+    APIVersion = case wrq:get_req_header("X-Ops-Server-API-Version", Req) of
+                     undefined ->
+                         <<"-1">>;
```

Comment:
So the intent here (at least as written in the RFC) was that we'd report back the API version level used to service the request - which would only not exist if the request was not serviced b/c requested version was invalid.